### ISSUE TITLE
Tomcat for windows containers?

### ISSUE THREAD

**Issue by jparasha** [docker-library/tomcat#66]
Trying to write docker file to download Tomcat for windows containers.
Unfortunately, openJDK too is not available for Windows containers. 
Anyone, has created image for windows containers or can give insights on the issue?
Thanks.

**Comment by burhanuddinbrf** [comment]
https://github.com/burhanuddinbrf/Tomcat-on-windows-container.git

**Comment by yosifkit** [comment]
We do have jdk 8 images based on microsoft/windowsservercore ([`openjdk:8-jdk-windowsservercore`](https://github.com/docker-library/openjdk/blob/9745c87a15896ec558429a826e23926e721e4846/8-jdk/windows/windowsservercore/Dockerfile)) and microsoft/nanoserver ([`openjdk:8-jdk-nanoserver`](https://github.com/docker-library/openjdk/blob/9745c87a15896ec558429a826e23926e721e4846/8-jdk/windows/nanoserver/Dockerfile)).  They were merged in https://github.com/docker-library/openjdk/pull/88 and pushed to the Docker Hub with https://github.com/docker-library/official-images/pull/26.  It should be possible to install tomcat on top of one or both of those.

**Comment by burhanuddinbrf** [comment]
Yes you are right but most enterprise applications prefer oracle jdk rather than open-jdk .

**Comment by tianon** [comment]
Due to licensing, the official images cannot include Oracle's Java.  The
best we can do is OpenJDK, which is what the Linux variants of this repo
are already based on.  If we add Windows variants, they will be consistent
with that.

**Comment by burhanuddinbrf** [comment]
the idea was to install tomcat using chocolatey on the container.

**Comment by jparasha** [comment]
@yosifkit Yes finally , we have openJDK for windows. Though, it may now be possible to add Tomcat on top of it, still an official Tomcat image for windows will always be preferred and trusted by enterprises. Hope we get that support soon as well  :-)

@burhanuddinbrf Your contribution is great and does the job well. But, I will wait for official image as this is what being expected by client. Still, thanks for the suggestion. cheers!

**Comment by Babu2000** [comment]
**@burhanuddinbrf**  i have build tomcat image with your docker file with out any issues. i have ran container with tomcat image then container is exiting immediately. tomcat is not running.

**Comment by Babu2000** [comment]
**jparasha** Have you changed anything docker file?

**Comment by tianon** [comment]
Can you please move your discussion of that Docker image to an issue on its
repo instead of this one? Thanks 👍
